sdk/template/ci.yml

```diff
@@ -50,3 +50,5 @@ extends:
     Artifacts:
     - name: azure-template
       safeName: azuretemplate
+      # JRS - This will be removed prior to merge
```

Comment:
Remember to remove this.

Reply:
Oh yeah, it's gone. My test was successful.